src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -114,6 +114,22 @@ internal FileLocation CreateFileLocation(Uri uri, string uriBaseId)
             return fileLocation;
         }
 
+        internal FileLocation CreateFileLocation(PhysicalLocationVersionOne v1PhysicalLocation)
+        {
+            FileLocation fileLocation = null;
+
+            if (v1PhysicalLocation?.Uri != null)
+            {
+                fileLocation = new FileLocation
+                {
+                    Uri = v1PhysicalLocation.Uri,
+                    UriBaseId = v1PhysicalLocation.UriBaseId
+                };
+            }
+
+            return fileLocation;
```

Comment:
Doesn't this exist somewhere else?

Reply:
There is an overload that takes the uri and uribaseid

Reply:
ok.